release.sh

```diff
@@ -0,0 +1,91 @@
+set -e
+
+check_version () {
+	if ! [[ $1 =~ ^v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]
+	then
+	    echo -e "\nThat version does not match semver"
+	    exit
+	fi
+}
+
+get_branch () {
+	# Get the branch to release and abort if branch does not exist or is not checked out
+	echo "What branch do you want to release?"
+	read branch
+	git rev-parse --verify $branch
+	current_branch=`git rev-parse --abbrev-ref HEAD`
+
+	if ! [[ $current_branch = $branch ]]
```

Comment:
If we can only support release the current branch (which is fine, I think), maybe we should just show the user what the branch is, and ask them whether that's what they want.